ui/src/main/java/com/theoplayer/android/ui/SeekBar.kt

```diff
@@ -42,7 +45,12 @@ fun SeekBar(
     var wasPlayingBeforeSeek by remember { mutableStateOf(false) }
 
     Slider(
-        modifier = modifier,
+        modifier = modifier.systemGestureExclusion {
+            Rect(
+                topLeft = Offset(0f, 0f),
+                bottomRight = Offset(it.size.width.toFloat(), it.size.height.toFloat() + 20)
```

Comment:
Can you elaborate on the `+ 20`? Where does this number come from?

Reply:
Hello.
Thanks for pointing it out. Yes, you're right. In our case, we added this as a safe margin equal to the size of the material slider thumbnail. You may remove it. @MattiasBuelens